srcpkgs/lsof/template

```diff
@@ -9,38 +9,67 @@ hostmakedepends="perl which"
 short_desc="LiSt Open Files"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="custom:lsof"
-homepage="https://people.freebsd.org/~abe/"
-distfiles="https://fossies.org/linux/misc/lsof-${version}.tar.gz"
-checksum=a9865eeb581c3abaac7426962ddb112ecfd86a5ae93086eb4581ce100f8fa8f4
+homepage="https://github.com/lsof-org/lsof"
+distfiles="https://github.com/lsof-org/lsof/archive/${version}.tar.gz"
+checksum=8ff4c77736cc7d9556da9e2c7614cc4292a12f1979f20bd520d3c6f64b66a4d7
 
 pre_configure() {
 	export LSOF_CC="$CC"
 	export LSOF_CFGF="$CFLAGS"
 	export LSOF_CFGL="$LDFLAGS"
 	export LSOF_INCLUDE="${XBPS_CROSS_BASE}/usr/include"
 
-	vsed "s|/\* #define\tHASSECURITY\t1 \*/|#define\tHASSECURITY\t1|" \
+	vsed -e "s|/\* \(#define\tHASSECURITY\t1\) \*/|\1|" \
 		-i dialects/linux/machine.h
 }
 
 pre_check() {
 	# test resolves hostname
-	cp /etc/hosts ./hosts
+	cp /etc/hosts ./hosts.bak
 	printf '# +++ xbps-src +++ required for lsof tests\n127.0.0.100 %s\n' "$(hostname)" >> /etc/hosts
 	# write current config to test db
 	cd tests && ./Add2TestDB
 }
+
 do_check() {
-	make -C tests
+	cd tests
+	make
+	if [ "$XBPS_CHECK_PKGS" = full ]; then
+		make opt	# optional tests
+	fi
 }
+
 post_check() {
-	# cleanup polluted hosts file
-	mv ./hosts /etc/hosts
+	if [ "$XBPS_BUILD_ENVIRONMENT" = "void-packages-ci" ]; then
+		# Don't do anything, as trying to mv hosts.bak to /etc/hosts
+		# results in EBUSY return code on GitHub Actions, making the
+		# tests seem like they're failing.
+		:
+	else
+		# cleanup polluted hosts file
+		mv ./hosts.bak /etc/hosts
+	fi
```

Comment:
` if ! [ ... ]; then` would be a bit cleaner

Reply:
I did it like this so that I could explain why we didn't restore `/etc/hosts` in case of `void-packages-ci`

Reply:
_removed that part in the latest commit_